tests/test_opts.py

```diff
@@ -0,0 +1,170 @@
+import re
+from collections import defaultdict
+from collections import OrderedDict
+import sys
+sys.path.append('../user-query-optimizer')
+import optimizer
+import sqlparse
+
+def test_approximates(queries):
+    print("\n")
+    # dictionary from test-query-file -> list of line numbers in that query with an approx optimization
+    correct_opts = {
+        'test-query-3.txt': [0]
+    }
+    test_ops = {}
+
+    for ind, query in enumerate(queries):
+        # Instantiate optimizer object
+        schema = {} # Placeholder
+        op = optimizer.Optimizer(query, schema, "Presto")
```

Comment:
Please use some [testing fixtures](http://pythontesting.net/framework/pytest/pytest-fixtures-easy-example/) for these.